--- a/dlls/ntdll/fsync.c	2020-04-13 22:41:33.600616689 -0400
+++ a/dlls/ntdll/fsync.c	2020-04-19 07:33:23.466605725 -0400
## server_select takes 6 arguments not 4
@@ -1214,7 +1214,7 @@
 	user_apc_t apc;
     /* We have to make a server call anyway to get the APC to execute, so just
      * delegate down to server_select(). */
-    ret = server_select( NULL, 0, SELECT_INTERRUPTIBLE | SELECT_ALERTABLE, &zero );
+    ret = server_select( NULL, 0, SELECT_INTERRUPTIBLE | SELECT_ALERTABLE, &zero, NULL, NULL );
 
     /* This can happen if we received a system APC, and the APC fd was woken up
      * before we got SIGUSR1. poll() doesn't return EINTR in that case. The
